packages/material-ui-icons/src/utils/createSvgIcon.js

```diff
@@ -1,18 +1,20 @@
 import React from 'react';
 import SvgIcon from '@material-ui/core/SvgIcon';
 
-function createSvgIcon(path, displayName) {
-  let Icon = props => (
-    <SvgIcon {...props}>
-      {path}
-    </SvgIcon>
+export default function createSvgIcon(path, displayName) {
+  const Component = React.memo(
+    React.forwardRef((props, ref) => (
+      <SvgIcon {...props} data-mui-test={`${displayName}Icon`} ref={ref}>
```

Comment:
Find by displayName is not supported for memo components. Mostly because react itself ignores `displayName` on memo components.